src/core/xlio_extra.h

```diff
@@ -641,4 +645,163 @@ static inline struct xlio_api_t *xlio_get_api()
+                            struct sockaddr *__from, socklen_t *__fromlen);
+#endif
+
+ssize_t xlio_write(int __fd, __const void *__buf, size_t __nbytes);
+
+ssize_t xlio_writev(int __fd, const struct iovec *iov, int iovcnt);
+
+ssize_t xlio_send(int __fd, __const void *__buf, size_t __nbytes, int __flags);
+
+ssize_t xlio_sendmsg(int __fd, __const struct msghdr *__msg, int __flags);
+
+int xlio_sendmmsg(int __fd, struct mmsghdr *__mmsghdr, unsigned int __vlen, int __flags);
+
+ssize_t xlio_sendto(int __fd, __const void *__buf, size_t __nbytes, int __flags,
+                    const struct sockaddr *__to, socklen_t __tolen);
+
+ssize_t xlio_sendfile(int out_fd, int in_fd, off_t *offset, size_t count);
+
+ssize_t xlio_sendfile64(int out_fd, int in_fd, __off64_t *offset, size_t count);
+
+const char *dbg_sprintf_fdset(char *buf, int buflen, int __nfds, fd_set *__fds);
+
+int xlio_select(int __nfds, fd_set *__readfds, fd_set *__writefds, fd_set *__exceptfds,
+                struct timeval *__timeout);
+
+int xlio_pselect(int __nfds, fd_set *__readfds, fd_set *__writefds, fd_set *__errorfds,
+                 const struct timespec *__timeout, const sigset_t *__sigmask);
+int xlio_poll(struct pollfd *__fds, nfds_t __nfds, int __timeout);
+
+#if defined HAVE___POLL_CHK
+int xlio___poll_chk(struct pollfd *__fds, nfds_t __nfds, int __timeout, size_t __fdslen);
+#endif
+
+int xlio_ppoll(struct pollfd *__fds, nfds_t __nfds, const struct timespec *__timeout,
+               const sigset_t *__sigmask);
+
+#if defined HAVE___PPOLL_CHK
+int xlio___ppoll_chk(struct pollfd *__fds, nfds_t __nfds, const struct timespec *__timeout,
+                     const sigset_t *__sigmask, size_t __fdslen);
+#endif
+
+int xlio_epoll_create(int __size);
+
+int xlio_epoll_create1(int __flags);
+
+int xlio_epoll_ctl(int __epfd, int __op, int __fd, struct epoll_event *__event);
+
+int xlio_epoll_wait(int __epfd, struct epoll_event *__events, int __maxevents, int __timeout);
+
+int xlio_epoll_pwait(int __epfd, struct epoll_event *__events, int __maxevents, int __timeout,
+                     const sigset_t *__sigmask);
+int xlio_socketpair(int __domain, int __type, int __protocol, int __sv[2]);
+
+int xlio_pipe(int __filedes[2]);
+
+int xlio_open(__const char *__file, int __oflag, ...);
+
+int xlio_creat(const char *__pathname, mode_t __mode);
+
+int xlio_dup(int __fd);
+
+int xlio_dup2(int __fd, int __fd2);
+
+#ifdef _CHANGE_CLONE_PROTO_IN_SLES_10_
+int xlio_clone(int (*__fn)(void *), void *__child_stack, int __flags, void *__arg);
+#endif
+
+pid_t xlio_fork(void);
+
+pid_t xlio_vfork(void);
+
+int xlio_daemon(int __nochdir, int __noclose);
+
+int xlio_sigaction(int signum, const struct sigaction *act, struct sigaction *oldact);
+
+sighandler_t xlio_signal(int signum, sighandler_t handler);
+
+#if defined(DEFINED_NGINX)
+int xlio_setuid(uid_t uid);
+
+pid_t xlio_waitpid(pid_t pid, int *wstatus, int options);
+#endif // DEFINED_NGINX
```

Comment:
Remove this block. NGINX doesn't use XLIO explicitly.

Reply:
Done